### PR TITLE
feat: add stderr.log to error_log

### PR DESCRIPTION
**Pull request by fengmk2**
##### Checklist

- [x] `npm test` passes
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message follows commit guidelines

### CODE REVIEW

File: config/config.default.js [hunk 1]
```diff
@@ -21,9 +21,10 @@ module.exports = appInfo => {
     appid: '',
     secret: '',
     cmddir: path.dirname(require.resolve('commandx/package.json')),
-    logdir,
+    logdir: alinodeLogdir,
     error_log: [
       path.join(appRoot, `logs/${appInfo.pkg.name}/common-error.log`),
+      path.join(appRoot, 'logs/stderr.log'),
```
Comment (fengmk2) [inline config/config.default.js:27]: @JacksonTian 如果这个 stderr.log 文件不存在， agentx 会抛错吗？

**Comment by popomore** [comment]
+1

**Comment by popomore** [comment]
发个版本

**Comment by fengmk2** [comment]
好。

**Comment by fengmk2** [comment]
1.1.0

File: config/config.default.js (hunk 1 above)
Comment (JacksonTian) [reply config/config.default.js:27]: no。

**Pull request merged**